irohad/main/impl/on_demand_ordering_init.cpp

```diff
@@ -132,9 +132,11 @@ namespace iroha {
           // with number of peers
           auto &peer =
               current_peers_[permutation[reject_round % permutation.size()]];
-          log_->debug("For round {}, using OS on peer: {}",
-                      consensus::Round{current_round.block_round, reject_round},
-                      peer->toString());
+          log_->debug(
+              "For {}, using OS on peer: {}",
+              consensus::Round{current_round.block_round + block_round_advance,
```

Comment:
I thought, isn't it wasteful to create a struct just to place it into the log? Maybe do a manual output here?

Reply:
I think we can use this approach for simplicity now, and optimize it later if it turns out to be a bottleneck.